burn-slashed-collateral.md

```diff
@@ -0,0 +1,123 @@
+
+Confiscated collateral is currently redistributed to the nodes that solved a data request honestly. The problem with that approach is that it creates an obvious incentive for malicious nodes operators to perform several types of collateral-stealing attacks on other witnesses. Two types of attacks are described in [the discussion](burn-slashed-collateral) that lead to this WIP.
+
+The first and easiest to execute attack revolves around creating a data request with 'inconsistent' sources. This goal here is for a number of witnesses to reveal an out-of-consensus value such that the quorum requirements are still satisifed. This results in a number of coins (`liars * collateral`) being slashed and redistributed amongst honest nodes. The attacker does not directly benefit from creating this specific data request because he has no control over which nodes earn the redistributed collateral. However, if he repeats this attack often enough and controls a non-negligible part of the ARS his nodes will statistically gain more Witnet coins than if the attack was not taking place due to his nodes receiving part of the slashed coins.
+
+The second attack is a bit more involved. Instead of targetting random nodes with a malicious data request, the attacker first builds a mapping of public keys to IP addresses. Once nodes have been identified he waits until his node is selected to solve a data request. At the reveal stage of the data request, he will then launch a DDoS attack on some of the other witnessing nodes such that the data request still meets the quorum demands. This will result in the attacked nodes being unable to reveal their answer and they will get slashed. As the attacker was also chosen to solve this data request, he directly benefits from this data request since he will receive part of the slashed collateral.
+
+The best solution to mitigate this type of attacks is to render them economically uninteresting. This can be achieved through burning the slashed collateral instead of redistributing it amongst honest nodes. Of course it is still possible to execute these attacks and purposely burn collateral, but by taking away the economic incentive, they are less likely to occur.
+
+## Specification
+
+Instead of distributing slashed collateral amongst all honest revealers, the slashed collateral is sent to a burn address. The new `calculate_witness_reward` function needs to be rewritten such that instead of a slashed collateral remainder, it returns the witness reward and total amount of burned collateral:
```

Comment:
```suggestion
Instead of distributing slashed collateral amongst all honest revealers, the slashed collateral MUST be sent to a burn address. The new `calculate_witness_reward` function MUST be rewritten such that instead of a slashed collateral remainder, it returns the witness reward and total amount of burned collateral:
```